openquakeplatform_building_class/templates/building-class/building-class-not-auth.html

```diff
@@ -64,9 +63,3 @@ <h1>{% trans "Building Classification Survey" %}</h1>
   </div>
   <div id="dialog-message" title="Error" style="display: none; width: 600px;"></div>
 {% endblock body %}
-
-{% block extra_script %}
-    {{block.super}}
-    {% include 'search_scripts.html' %}
```

Comment:
Remove just the include

Reply:
Fixed with this 828d81e0221f19983e8848ea39ebd72625749c56